sbi/neural_nets/embedding_nets/resnet.py

```diff
@@ -0,0 +1,321 @@
+from functools import partial
+from typing import Callable, Type
+
+import torch
+import torch.nn as nn
+
+
+def zero_pad(x: torch.tensor, c_out: int) -> torch.tensor:
```

Comment:
type hints for tensor either as `torch.Tensor` or at best importing `from torch import Tensor` and using `Tensor`